scene/3d/xr_hand_modifier_3d.cpp

```diff
@@ -111,22 +111,30 @@ void XRHandModifier3D::_get_joint_data() {
 		joints[i].parent_joint = -1;
 	}
 
-	Skeleton3D *skeleton = get_skeleton();
+	const Skeleton3D *skeleton = get_skeleton();
 	if (!skeleton) {
 		return;
 	}
 
-	XRServer *xr_server = XRServer::get_singleton();
+	const XRServer *xr_server = XRServer::get_singleton();
 	if (!xr_server) {
 		return;
 	}
 
-	Ref<XRHandTracker> tracker = xr_server->get_hand_tracker(tracker_name);
-	if (tracker.is_null()) {
+	const Ref<XRHandTracker> tracker = xr_server->get_tracker(tracker_name);
+	if (!tracker.is_valid()) {
```

Comment:
```suggestion
	if (tracker.is_null()) {
```
Let's keep this using the cleaner form

Reply:
Indeed - switched to the cleaner form.